include/fast_float/ascii_number.h

```diff
@@ -32,7 +32,7 @@ template <typename UC> fastfloat_really_inline constexpr bool has_simd_opt() {
 // able to optimize it well.
 template <typename UC>
 fastfloat_really_inline constexpr bool is_integer(UC c) noexcept {
-  return !(c > UC('9') || c < UC('0'));
+  return static_cast<uint8_t>(c - UC('0')) < 10;
```

Comment:
This will overflow when `sizeof(UC) > sizeof(uint8_t)` resulting in false positives for:

```c++
wchar_t ch = '0';
ch += 256;
is_integer(ch); // incorrectly true
```

Reply:
> This will overflow when `sizeof(UC) > sizeof(uint8_t)` resulting in false positives for:
> 
> ```c++
> wchar_t ch = '0';
> ch += 256;
> is_integer(ch); // incorrectly true
> ```

How does it look now?

Reply:
Alright it should be good now.